### PR TITLE
Swift 4.2 / Xcode 10 update

### PR DESCRIPTION
**Pull request by cdillard**

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -9,4 +9,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d5607466d3d3e3deae872088ea7e172451978d1e
 
-COCOAPODS: 1.2.1
+COCOAPODS: 1.4.0
```
Comment (nhgrif) [inline Podfile.lock]: Isn't 1.5.3 current?  Should we run `bundle update cocoapods`?

File: MetovaTestKit.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -612,13 +612,16 @@
 			files = (
 			);
 			inputPaths = (
+				"${PODS_PODFILE_DIR_PATH}/Podfile.lock",
```
Comment (nhgrif) [inline MetovaTestKit.xcodeproj/project.pbxproj:599]: What's this?

**Review by nhgrif — CHANGES_REQUESTED** [review]

**Comment by nhgrif** [comment]
I changed this to merge into develop instead of master.

File: MetovaTestKit.xcodeproj/project.pbxproj (hunk 2 above)
Comment (cdillard) [reply MetovaTestKit.xcodeproj/project.pbxproj:599]: I am not sure. Xcode 10 added this maybe ? or pod install

**Review by nhgrif — CHANGES_REQUESTED** [review]
The only remaining thing that needs to be changed, I think, is the swift_version in the podspec.  Sorry I missed this yesterday.

**Review by nhgrif — APPROVED** [review]

**Pull request merged**